### PR TITLE
Behat - Create story row paragraph and verify

### LINKED ISSUE #1160: Behat test - Create story row paragraph and verify
- Create a story row paragraph and add it in testing landing page and verify the content.

### PR DESCRIPTION
**Pull request by pradip3271**
Fixes #1160 ...

## Changes proposed in this PR

- Create test landing page
- Add story row paragraph
- Verify story row paragraph content
- Delete test landing page

### CODE REVIEW

File: tests/behat/features/bootstrap/DrupalCRFeatureContext.php [hunk 1]
```diff
@@ -483,4 +483,63 @@ public function iClickOnElement(string $field): void
...
+  /**
+   * Creates landing page with story row paragraphs with fields title, fundraiser_total,
+   *         fundraiser_copy, fundraiser_img, fundraiser_bg_color, beneficiary_copy, beneficiary_img
+   *
+   * @Given I create a :type( page ) with :title( title) and story row paragraph with following fields:
+   */
+  public function createLandingPageWithStoryRowParagraph( $type, $title,TableNode $fields ) {
+
+    // First, create a landing page node.
+    $node = (object) [
```
Comment (Saphyel) [inline tests/behat/features/bootstrap/DrupalCRFeatureContext.php]: I don't like this hardcode way to test things... could you reuse any other existing node instead?

File: tests/behat/features/landing-page.feature [hunk 2]
```diff
@@ -1,4 +1,4 @@
-@wip
+
 Feature: Landing-page
```
Comment (Saphyel) [inline tests/behat/features/landing-page.feature]: why an empty line at the top?

File: tests/behat/features/bootstrap/DrupalCRFeatureContext.php (hunk 1 above)
Comment (pradip3271) [inline tests/behat/features/bootstrap/DrupalCRFeatureContext.php]: @Saphyel  existing node will be deleted once test scenario completed.

File: tests/behat/features/landing-page.feature (hunk 2 above)
Comment (pradip3271) [inline tests/behat/features/landing-page.feature]: @Saphyel  removed

File: tests/behat/features/bootstrap/DrupalCRFeatureContext.php [hunk 3]
```diff
@@ -483,4 +483,63 @@ public function iClickOnElement(string $field): void
     $this->getSession()->getPage()->find('css', $field)->click();
   }
 
+  /**
+   * Creates landing page with story row paragraphs with fields title, fundraiser_total,
+   *         fundraiser_copy, fundraiser_img, fundraiser_bg_color, beneficiary_copy, beneficiary_img
+   *
+   * @Given I create a :type( page ) with :title( title) and story row paragraph with following fields:
+   */
+  public function createLandingPageWithStoryRowParagraph( $type, $title,TableNode $fields ) {
```
Comment (Saphyel) [inline tests/behat/features/bootstrap/DrupalCRFeatureContext.php]: can you improve the code style here ?

**Pull request merged**